Model/User.php

```diff
@@ -120,7 +120,6 @@
      */
     public function __construct()
     {
-        $this->salt = base_convert(sha1(uniqid(mt_rand(), true)), 16, 36);
```

Comment:
Well, we don't care whether it is null or no just after instantiation. Providing a value on instantiation is not broken. It is just useless (as we will never use this value to hash a password).